src/main/java/gomedic/commons/util/StringUtil.java

```diff
@@ -27,15 +28,15 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         requireNonNull(sentence);
         requireNonNull(word);
 
-        String preppedWord = word.trim();
+        String preppedWord = word.trim().toLowerCase();
         AppUtil.checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
         AppUtil.checkArgument(preppedWord.split("\\s+").length == 1, "Word parameter should be a single word");
 
-        String preppedSentence = sentence;
+        String preppedSentence = sentence.toLowerCase();
         String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
 
         return Arrays.stream(wordsInPreppedSentence)
-                .anyMatch(preppedWord::equalsIgnoreCase);
+                .anyMatch(x -> x.contains(preppedWord));
     }
 
     /**
```

Comment:
What's the rational changing equalsIgnoreCase and just to make it to lowercase ?